docs/src/main/paradox/scala/http/migration-guide/migration-from-spray.md

```diff
@@ -103,7 +103,7 @@ ToResponseMarshaller.oneOf(
 }
 ```
 
-After:
+After: 
```

Comment:
I wrote my thoughts as a comment. But I had to edit the file in order to write a comment